File: 62/aux.py
from solve import solve


def main():
    for index, test in enumerate(get_tests()):
        n, m = test
        print("Test %d\n Give a %d x %d matrix, ways to reach lower-right corner: %d\n" % (index, n, m, solve(n, m)))


def get_tests():
    """
    format (N, M)
    """
    return [
        (2, 5),
        (2, 2),
        (5, 5),
        (5, 1)
    ]


if __name__ == '__main__':
    main()

File: 07/aux.go
package main

import "fmt"

func main() {

	for index, test := range getTests() {
		fmt.Printf(
			"Test %d\nFor message %s, there are %d ways to decode it\n\n",
			index, test, solve(test),
		)
	}
}

func getTests() []string {
	return []string{
		"111",
		"261341",
		"1111",
		"1111111",
		"4",
		"34",
		"26",
		"201",
		"161203",
	}
}
